code/datums/elements/snail_crawl.dm

```diff
@@ -31,5 +31,5 @@
 
 	var/turf/open/OT = get_turf(snail)
 	if(istype(OT))
-		OT.MakeSlippery(TURF_WET_LUBE, 20)
+		OT.MakeSlippery(TURF_WET_WATER, 2.5 SECONDS) //SKYRAT EDIT: Roundstart Snails - No more lube
```

Comment:
Unsure why we'd make it last even longer, 2 seconds is still kind of a while, especially with server lag taken into account. Not a hill I'm going to die on, but y'know.